webapp/channels/src/components/widgets/icons/fa_warning_icon.tsx

```diff
@@ -8,21 +8,17 @@ import LocalizedIcon from 'components/localized_icon';
 import {t} from 'utils/i18n';
 
 type Props = {
-    additionalClassName: string | null;
+    additionalClassName?: string;
 }
 
-export default class WarningIcon extends React.PureComponent<Props> {
-    public static defaultProps: Props = {
-        additionalClassName: null,
-    };
-
-    public render(): JSX.Element {
-        const className = 'fa fa-warning' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
-        return (
-            <LocalizedIcon
-                className={className}
-                title={{id: t('generic_icons.warning'), defaultMessage: 'Warning Icon'}}
-            />
-        );
-    }
+function WarningIcon(props: Props) {
+    const className = 'fa fa-warning' + (props.additionalClassName ? ' ' + props.additionalClassName : '');
```

Comment:
I think we can use here `classNames` as `classNames('fa fa-warning', props.additionalClassName)` and should do the same. Do you mind trying?

Reply:
I've changed it to `classNames` and I a little bit changes the style to destruct the props like:
```
const WarningProps = ({additionalClassName}:Props) => {
...
}
```
It's more common for me and looking `webapp/channels/src/components/copy_text.tsx` I guess it's convention here.